### PR TITLE
Add rss reading example

### PR DESCRIPTION
**Pull request by kadler**

### CODE REVIEW

File: non-wheel/rss-example/rss-example.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+from sys import argv
+
+url = 'http://ibmsystemsmag.com/CMSTemplates/IBMSystemsMag/Feeds/Open-Your-i.aspx'
+if len(argv) > 1:
+    url = argv[1]
+
+feed = feedparser.parse(url)
+
+for entry in feed['entries'][:5]:
+    print("\"{e[title]}\"\n{e[link]}\n".format(e=entry))
```
Comment (jordiwes) [inline non-wheel/rss-example/rss-example.py]: When I run this I get:
Traceback (most recent call last):
  File "./rss-example.py", line 14, in <module>
    print("\"{e[title]}\"\n{e[link]}\n".format(e=entry))
UnicodeEncodeError: 'latin-1' codec can't encode character '\u2019' in position 4: ordinal not in range(256)

**Comment by jbh** [comment]
Any reason this isn't merged yet?

**Comment by kadler** [comment]
Just needs to be reviewed and tested. @jordiwes had been doing that previously.

**Comment by jordiwes** [comment]
See unicode error above... if we feel fine to merge anyways, it's on you ;)

**Comment by kadler** [comment]
What unicode error?

**Comment by jordiwes** [comment]
Wow, I never pressed the submit button. Sorry

File: non-wheel/rss-example/rss-example.py (hunk 1 above)
Comment (kadler) [reply non-wheel/rss-example/rss-example.py]: That's weird. [U+2019](https://www.fileformat.info/info/unicode/char/2019/index.htm) is the right-single quote character, but there's not even a regular single quote on line 14. I can't recreate the problem (even in QSH/QP2TERM).

The other interesting thing is the line it prints out:
```python
print(""{e[title]}"\n{e[link]}\n".format(e=entry))
```
What happened to the backslash? It should be `print("\"...`, not `print(""....`.

How did you test it?

**Pull request closed without merging**